experiments/download_ssl4eo.py

```diff
@@ -587,10 +471,11 @@ def worker(idx: int) -> None:
 
     # set indices
     if args.indices_range is not None:
-        indices = range(args.indices_range[0], args.indices_range[1])
+        indices = list(range(args.indices_range[0], args.indices_range[1]))
```

Comment:
```suggestion
        indices = list(range(*args.indices_range))
```
Does it have to be a list? This will be a pretty large list.

Reply:
Not a must, i think I made it a list to be consistent with the other option.
```
# set indices
if args.indices_range is not None:
    indices = list(range(args.indices_range[0], args.indices_range[1]))
else:
    indices = []
    for key in match_coords.keys():
        indices.append(int(key))
```